tests/unittests/test_net_freebsd.py

```diff
@@ -1,8 +1,27 @@
-from cloudinit import net
+import cloudinit.net
+import cloudinit.net.network_state
+
+
+from cloudinit.tests.helpers import (CiTestCase, mock, readResource, dir2dict)
+
+
+import os
+import yaml
```

Comment:
Nit: from the second bullet of the [style guide](https://www.python.org/dev/peps/pep-0008/#imports)

```suggestion
import os
import yaml

import cloudinit.net
import cloudinit.net.network_state
from cloudinit.tests.helpers import (CiTestCase, mock, readResource, dir2dict)
```